pkg/config/crd/store.go

```diff
@@ -51,6 +51,12 @@ const (
 	// initial data since it may arrive in any order, and it
 	// would result in temporary invalid status.
 	initializationPeriod = time.Second * 2
+
+	// crdRetryTimeout is the timeout duration to retry initialization
+	// of the caches when some CRDs are missing. The actual timeout
+	// can be customized through the timeout for the context passed to
+	// Init() method.
+	crdRetryTimeout = time.Minute * 5
```

Comment:
Do we have data on timing of CRDs? 5 minutes seems like a very very long time to wait. Keep in mind that we do this at Mixer startup before we start serving traffic.